### ISSUE TITLE
getting sporadic 401 responses

### ISSUE THREAD

**Issue by npomfret** [couchbaselabs/react-native-couchbase-lite#19]
My application is receiving intermittent 401 errors from CBLite.  Sometimes just one or two, but eventually 100% of requests fail (and it seems to causes very high CPU load).

Do I need to refresh the session periodically?

```
{ _bodyInit: '',
  _bodyText: '',
  type: 'default',
  url: 'http://admin:pass@localhost:5984/myapp/-7B-cilWnWogJzIQ_xTsLYP',
  status: 401,
  ok: false,
  statusText: undefined,
  headers: 
   { map: 
      { 'accept-ranges': [ 'bytes' ],
        'content-length': [ '0' ],
        'www-authenticate': [ 'Digest realm="CouchbaseLite", qop="auth", nonce="4D0A64E6-84AA-40E1-A8E0-F2CC8A524396"' ],
        date: [ 'Thu, 03 Mar 2016 10:06:12 GMT' ] } } }

```

**Comment by ssomnoremac** [comment]
again, if you're updating you need look at my answer for #21

**Comment by jamesnocentini** [comment]
Looks like this was fixed in #21. Please re-open if necessary.
